covalent/_results_manager/results_manager.py

```diff
@@ -69,12 +64,12 @@ def get_result(dispatch_id: str, wait: bool = False) -> Result:
         )
         result_object = pickle.loads(codecs.decode(result["result"].encode(), "base64"))
 
-    except MissingLatticeRecordError as e:
-        app_log.warning(
-            f"Dispatch ID {dispatch_id} was not found in the database. Either the Dispatch ID is incorrect or wait a couple of seconds before trying again."
+    except MissingLatticeRecordError:
+        app_log.debug(
+            f"Dispatch ID {dispatch_id} was not found in the database. Incorrect dispatch id."
```

Comment:
Assuming the `404` is returned correctly from the server then this actually _should_ be a warning.

Reply:
Fixed, thanks